widgets/range-slider/range-slider.js

```diff
@@ -95,6 +94,7 @@ function rangeSlider({
         templates
       });
 
+      var Slider = autoHide(headerFooter(require('../../components/Slider/Slider')));
```

Comment:
Ahah that's some kind of horrible thing required by tests and linked to what I told you about moving decorators declarations to the index.js file (searchBox = autoHide(require('./widgets/searchBox))).

In (widgets) tests we need to mock autoHide etc.. Because they are part of the widget requires.

And to do so, we need to mock them BEFORE the render function is called. So we need to require them only where needed.